platform/config/service/mvc/gateway-svc.yml.in

```diff
@@ -68,31 +68,10 @@ spec:
                     name: gateway-persistent-storage
                 livenessProbe:
                     httpGet:
-                        path: /
+                        path: /v1/ping
```

Comment:
As discussed, the CPU limit should be removed or increased.

Reply:
This is fixed.